### ISSUE TITLE
mat-expansion-panel with animations does not not respect first expandedHeight and collapsedHeight values from an observable

### ISSUE THREAD

**Issue by biltongza** [angular/components#15944]
#### What is the expected behavior?
mat-expansion-panel should respect the heights given to it on init.

#### What is the current behavior?
The heights only apply once the expansion panel is expanded.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-erjhve

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 7.2.14, Material 7.3.7, Windows 10 1809, TS 3.2.4, Chrome 74.0.3729.131, Firefox 66.0.3 64 bit

## Is there anything else we should know?
Using `NoopAnimationsModule` appears to fix the issue, but I want animations.

**Comment by biltongza** [comment]
I have found a disgusting workaround: Have two `mat-expansion-panel-headers` with an `ngIf` on each of them. One has the standard height, one has the alternative height. For example:
```
<mat-expansion-panel-header *ngIf="mobileView$ | async"
    [expandedHeight]="mobileContentHeight"
    [collapsedHeight]="mobileContentHeight"
    fxLayout="row"
    fxLayoutAlign="space-around center"
>
    <ng-container *ngTemplateOutlet="content1; context: { row: row, headers: headers }">
    </ng-container>
</mat-expansion-panel-header>

<mat-expansion-panel-header *ngIf="!(mobileView$ | async)"
    fxLayout="row"
    fxLayoutAlign="space-around center"
>
    <ng-container *ngTemplateOutlet="content2; context: { row: row, headers: headers }">
    </ng-container>
</mat-expansion-panel-header>
```

🤢

**Comment by meepou** [comment]
I was about to post the same issue when I saw yours, I had a stackblitz as well : https://stackblitz.com/edit/angular-p6rpru-nrrbxk (just refresh the preview and you see accordions jump)

I can't find a css workaroud as it happens so fast, if someone has a temporary solution ...

Angular 7x, Material 7x

**Comment by biltongza** [comment]
This appears to have been fixed in 8.0.2 but was not noted in the release notes.
